operator/internal/config/options.go

```diff
@@ -19,6 +21,7 @@ import (
 // from a provided configuration file.
 func LoadConfig(scheme *runtime.Scheme, configFile string) (*configv1.ProjectConfig, *TokenCCOAuthConfig, ctrl.Options, error) {
 	options := ctrl.Options{Scheme: scheme}
+	leaderElection := leaderelection.LeaderElectionDefaulting(openshiftconfigv1.LeaderElection{}, "", "")
```

Comment:
I think I would move this down closer to where it is used and maybe call it `leaderElectionDefaults` to make it more expressive.